pkg/validator/ini.go

```diff
@@ -0,0 +1,17 @@
+package validator
+
+import (
+	"gopkg.in/ini.v1"
+)
+
+type IniValidator struct{}
+
+// Validate implements the Validator interface by attempting to
+// parse a byte array of ini
+func (iv IniValidator) Validate(b []byte) (bool, error) {
+	_, err := ini.LoadSources(ini.LoadOptions{}, b)
+	if err != nil {
+		return false, err
```

Comment:
The error output needs to include the line and column number. For example:

```
Error at line 1 column 3:
```

Since ini does not seem to output that by default you'll need to calculate it. Look at how that's being done in the [json validator](https://github.com/Boeing/config-file-validator/blob/main/pkg/validator/json.go#L14)

Reply:
Created #26 for helping to enforce this in the future

Reply:
I didn't find possibility to calculate line&column
Checked another go ini solutions
e.g. [ini/parser](https://github.com/gookit/ini/tree/master/parser/) 
meanwhile pr set to draft mode

Reply:
Wrote an [enhancement request](https://github.com/go-ini/ini/issues/341) to go-init to add this output to the error message.